std_msgs/Header header

builtin_interfaces/Time time           # All observations should be at this time (UTC inc. leap seconds)
Observation[] obs   # Vector of observations

